### DISCUSSION TITLE
Load the local map service using itwin.js

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#2327] [Q&A] [answered]
I am using itwin.js to load the local map service. The same WMTs URL can display the map image in cesium.

When I use the control in itwin.js to load the same URL, I always report an error http://localhost:8010/WeServer/wmts?Service=WMTS&Request=GetCapabilities

I want know if itwin.js now supports loading local map services?
 If support, what are the correct steps? thank you
![1](https://user-images.githubusercontent.com/26344907/134305301-b07c6976-56c5-4f8b-a8b0-60a2d78d80f9.PNG)
.
![2](https://user-images.githubusercontent.com/26344907/134305345-4324ce81-78f9-4d17-b434-0050a751e56c.PNG)
![3](https://user-images.githubusercontent.com/26344907/134305369-0440e04e-709d-49c7-8180-0e2418a72715.PNG)

**Comment by pmconne** [comment, marked as answer]
@mdastous-bentley

**Reply by mdastous-bentley** [reply]
Hi, does your service has a TileMatrix using EPPSG:3857 and is compatible with GoogleMaps tiling ?  Also, It might be useful to attach the GepCapabilities response as a text file.

Thanks

**Reply by shenshengyi** [reply]
OK，Thanks.
